Sources/MapboxNavigation/RecentItem.swift

```diff
@@ -2,21 +2,25 @@ import Foundation
 import CarPlay
 
 /**
- Struct, which represents recently found search item on CarPlay.
+ Struct, which represents recently found search item on CarPlay. When storing recent items in an array
+ use dedicated methods for addition: `[RecentItem].add(_:)`, and removal: `[RecentItem].remove(_:)`.
  */
 public struct RecentItem: Equatable, Codable {
     
     /**
      Property, which contains information regarding geocoder result.
      */
-    public var navigationGeocodedPlacemark: NavigationGeocodedPlacemark
+    public private(set) var navigationGeocodedPlacemark: NavigationGeocodedPlacemark
 
     var timestamp: Date
     
-    static var filePathUrl: URL {
+    static var recentItemsPathUrl: URL? {
```

Comment:
While you’re at it, “URL” [should be capitalized](https://swift.org/documentation/api-design-guidelines/#follow-case-conventions) in the name when it isn’t the first word in the name.